### PR TITLE
add getJobStatesMap testCase

### PR DESCRIPTION
**Pull request by deepikasl**
https://github.com/Shippable/bat/issues/406

screenshot from 2017-03-13 12 16 22

### CODE REVIEW

File: tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js [hunk 1]
```diff
@@ -190,6 +189,32 @@ describe('Project History',
           }
         );
 
+        it('Get JobStatesMap',
+          function (done) {
+            this.timeout(0);
+
+            var query = util.format('projectIds=%s',
+              projectId);
+            shippable.getJobStateMaps(query,
```
Comment (shrivara) [inline tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js]: What is the test case here? Just that route works okay without errors?

Comment (deepikasl) [inline tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js]: Yes.

File: tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js [hunk 2]
```diff
@@ -190,6 +189,32 @@ describe('Project History',
           }
         );
 
+        it('Get JobStatesMap',
+          function (done) {
+            this.timeout(0);
+
+            var query = util.format('projectIds=%s',
```
Comment (shrivara) [inline tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js]: This is under 80 characters if the lines are merged.

File: tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js [hunk 3]
```diff
@@ -190,6 +189,32 @@ describe('Project History',
...
+              projectId);
+            shippable.getJobStateMaps(query,
+              function (err, jobStatesMaps) {
```
Comment (shrivara) [inline tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js]: `jobStatesMap` isn't used anywhere. Can be removed.

File: tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js [hunk 4]
```diff
@@ -190,6 +189,32 @@ describe('Project History',
...
+                if (err) {
+                  isTestFailed = true;
+                  var testCase =
+                    util.format(
```
Comment (shrivara) [inline tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js]: Please add `util` to `predef` in `package.json`. This is showing up as an error in jshint.

Comment (deepikasl) [inline tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js]: util is already there in `dependencies` in `package.json`

File: tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js [hunk 5]
```diff
@@ -190,6 +189,31 @@ describe('Project History',
...
+                      '\n - [ ] %s Get JobStatesMap for projectId:' +
+                      ' %s failed with error: %s', testSuiteDesc, projectId,
+                      err);
+                  testCaseErrors.push(testCase);
+                  assert.equal(err, null);
+                  return done();
+                } else {
```
Comment (shrivara) [inline tests/organization-owner/github/5-projects/1.2.08.ProjectHistory.js]: Redundant return done(); We can use the same one in both places. Rest looks good.

**Pull request merged**